src/App.css

```diff
@@ -59,3 +82,165 @@ body {
     transform: rotate(360deg);
   }
 }
+
+.action {
+  width: 4.063rem;
+  height: 1.125rem;
+  margin: 0 1.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.1rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  padding: 0 0 0 1.2%;
+}
+
+.title {
+  width: 20.313rem;
+  height: 1.813rem;
+  margin: 0.488rem 20.188rem 0 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+  padding: 0 0 0 1.2%;
+}
+
+.author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  padding: 0 0 0 1.2%;
+}
+
+.comment {
+  width: 4.563rem;
+  height: 1.198rem;
+  margin: 1.188rem 0.938rem 0.125rem 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.999rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background-color: #fff;
+  border: none;
+}
```

Comment:
- [ ] **[OPTIONAL]** I suggest you add CSS property `cursor: pointer;` to the `Remove` button to improve user experience and interaction on your page.